src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_FindReferences_Legacy.cs

```diff
@@ -62,19 +59,33 @@ public static partial class SymbolFinder
         /// information as the search is undertaken.</param>
         /// <param name="documents">An optional set of documents to be searched. If documents is null, then that means "all documents".</param>
         /// <param name="cancellationToken">An optional cancellation token.</param>
-        public static async Task<IEnumerable<ReferencedSymbol>> FindReferencesAsync(
+        public static Task<IEnumerable<ReferencedSymbol>> FindReferencesAsync(
             ISymbol symbol,
             Solution solution,
             IFindReferencesProgress progress,
             IImmutableSet<Document> documents,
             CancellationToken cancellationToken = default)
+        {
+            return FindReferencesAsync(
```

Comment:
This seems to be the only caller for the new overload (below). 
Some options to consider:
- remove the new overload (inlining it instead)
- make the new overload private

Reply:
turns out our testing harness also calls the new overload.  And, if we don't have that overload available to it, thne it just has to repeat the logic in it (making the progress/adapter objects).  So i'll leave as is, but comment that this is internal to let tests use this.